ror-tools/src/main/scala/tech/beshu/ror/tools/RorToolsApp.scala

```diff
@@ -246,7 +246,7 @@ trait RorTools {
 
   private lazy val esPathOption =
     opt[String]("es-path")
-      .text(s"Path to elasticsearch directory; default=${EsDirectory.defaultPath}")
+      .text(s"Path to elasticsearch directory; default=/usr/share/elasticsearch")
```

Comment:
The path is not correctly serialized on Windows. Even after fixing the format, the default Linux path does not make sense on Windows. So I hardcoded the Linux path in the message.